tile-server/src/main/java/index/PsqlSpatialIndexer.java

```diff
@@ -79,14 +91,25 @@ public void createMV(Canvas c, int layerId) throws Exception {
         String insertSql = "insert into " + bboxTableName + " values (";
         for (int i = 0; i < trans.getColumnNames().size() + 6; i ++)
             insertSql += "?, ";
+	if (isCitus) {
+	    insertSql += "?, ";
+	}
         insertSql += "ST_GeomFromText(?));";
+        System.out.println(insertSql);
         PreparedStatement preparedStmt = dbConn.prepareStatement(insertSql);
+	long start_ts = (new Date()).getTime();
```

Comment:
a little picky style thing - in Java, then naming convention of a variable is to capitalize the first letter of every word (other than the first), which is a bit different from C/C++. More info here: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html

Reply:
DONE/sorry.